README.md

```diff
@@ -1,2 +1,27 @@
-# Capstone-project_Concert
+# CAPSTONE PROJECT
 A page about an imaginary concert with my favorite singers and groups
+
+## DESCRIPTION
+In this project the CREATIVE COMMONS GLOBAL SUBMIT page will be replicated and the theme was changed to a imaginary concert in my country Bolivia.
+The main characteristics of the page were maintained as possible and the images and text content was changed according to the new theme chose.
+The main and the about page were built for desktop and mobile sizes.
+
+## BUILT WITH
+*Visual Studio Code*
+*Github Desktop application*
+*image and icon resources were collected from royalte free sites on the Internet*
+
+## HISTORY OF CHANGES
+**2022 May 16-18**: Version 1.0 released and presented for review.
+
+## LIVE DEMO: 
+
+Hans Zizold's Portfolio: https://hanszizold.github.io/Capstone-project_Concert/
+
+Author: Hans Paul Zizold
+    
+    GITHUB: https://github.com/HansZizold
+
+    Linkedin: https://www.linkedin.com/in/hans-paul-zizold-37129037/
+
```

Comment:
- [ ] Kindly add an acknowledgment section in your readme as previously instructed, the acknowledgment should have an appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).
- [ ] [OPTIONAL] It would be nice to add a Getting Started section describing the process of cloning and running the project.